packages/flutter_tools/lib/src/build_runner/resident_web_runner.dart

```diff
@@ -382,6 +382,34 @@ class ResidentWebRunner extends ResidentRunner {
     }
   }
 
+  @override
+  Future<void> debugTogglePlatform() async {
```

Comment:
I tried using this with Flutter for mobile just to learn about what this does, but it doesn't seem to do anything :man_shrugging: 

So, RSLGTM :)

Reply:
It changes scrolling semantics basically

Reply:
It didn't when I tested on mobile :thinking: